site/specs/numbers.json

```diff
@@ -765,77 +765,74 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves a list of available Npa-Nxx telephone numbers.</p>\n",
-        "operationId": "GET /accounts/{accountId}/availableNpaNxx",
+        "description": "<p>Retrieves a list of available Npa-Nxx telephone numbers.</p>",
+        "operationId": "GetAvailableNpaNxx",
+        "summary": "Retrieves available Npa-Nxx tns",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
             "in": "path",
             "required": true,
+            "style": "simple",
+            "explode": false,
+            "description": "User's account ID",
+            "example": "900000",
             "schema": {
-              "type": "string",
-              "default": null
-            },
-            "example": null
+              "type": "string"
+            }
           },
           {
             "name": "areaCode",
-            "description": "<p>The allowed number ranges are [2-9] for the first digit and [0-9] for both the second and third digits.</p>\n",
+            "description": "<p>The allowed number ranges are [2-9] for the first digit and [0-9] for both the second and third digits.</p>",
```

Comment:
Can you remove all `<p>` tags for single line descriptions

Reply:
sure